lib/vmpooler/statsd.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems' unless defined?(Gem)
+
+module Vmpooler
+  class Statsd
+    def initialize(
+      s = 'statsd',
+      port = 8125
+    )
```

Comment:
A nitpick and a must-do:

##### Imperative
Don't declare single letter variable names. Ever. Maybe in iterators, but definitely never in initializers.

##### Nitpick
The newlines in the required arguments don't help any. It can all be on one line.